homework/hw0.js

```diff
@@ -0,0 +1,72 @@
+startHomework();
```

Comment:
Ignore the scaffolding code (ex: `startHomework()`, `problem('name', ... )` ) in these files, all you have to care about is are comments with problem directions and the `// <Code Goes Here>` sections where you work through the answers! When you look at `homework.html` in your browser, you'll see your code run with pretty comments telling you where each homework / problem starts and ends 😃  We can chat about the quick scaffolding code I wrote later if you'd like.